### PR TITLE
Just in Time Infrastructure example

### PR DESCRIPTION
**Pull request by anovis**
#### Just in Time Infrastructure example
contains
- ec2
- vpc
- subnet
- security group
- instance profile

#### Bug Fix

- replace_value_nested_dict (works with list of strings now)
- find_value_nested_dict (works with list of strings now)
- aws_lookup (moved to apply so works with quasiparticle parents)

### CODE REVIEW

**Comment by anovis** [comment]
will add application that can get triggered in the userda

File: examples/quasiparticle/aws/jit/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# What is Just in Time Infrastructure
+Just in Time Infrastructure (JiTI) is a deployment model in which the entire infrastructure required for an application is written in code and deployed with the same life cycles alongside
+the application. Check our docs for more infomation on JiTI and the benifits this provides.
```
Comment (jacoblearned) [inline examples/quasiparticle/aws/jit/README.md]: `benifits` -> `benefits`

**Comment by bhl5kj** [comment]
LGTM

**Review by jacoblearned — CHANGES_REQUESTED** [review]
Small typos

File: examples/quasiparticle/aws/jit/README.md (hunk 1 above)
Comment (jacoblearned) [reply examples/quasiparticle/aws/jit/README.md]: As well as `infomation`-> `information`

**Review by jacoblearned — APPROVED** [review]

**Pull request merged**